engine-standalone-tracing/src/types/call_tracer.rs

```diff
@@ -117,6 +117,7 @@ impl CallTracer {
         self.call_stack.last_mut().unwrap().calls.push(frame);
     }
 
+    #[allow(clippy::missing_const_for_fn)]
```

Comment:
Why allow instead of changing the function to be const?

Reply:
Yes, it's an interesting case. But I guess it's because we call implicitly the [`deref_mut`](https://doc.rust-lang.org/std/vec/struct.Vec.html#method.deref_mut) function, which is not const.

Reply:
> Funny that clippy has a false positive for this case.

Yeah. It happens from time to time 😄